lib/matplotlib/quiver.py

```diff
@@ -779,142 +779,141 @@ def _h_arrows(self, length):
 
 
 _barbs_doc = r"""
-Plot a 2-D field of barbs.
+Plot a 2D field of barbs.
 
-Call signatures::
+Call signature::
+
+  barbs([X, Y], U, V, [C], **kw)
+
+Where *X*, *Y* define the barb locations, *U*, *V* define the barb
+directions, and *C* optionally sets the color.
+
+All arguments may be 1D or 2D. *U*, *V*, *C* may be masked arrays, but masked
+*X*, *Y* are not supported at present.
+
+Barbs are traditionally used in meteorology as a way to plot the speed
+and direction of wind observations, but can technically be used to
+plot any two dimensional vector quantity.  As opposed to arrows, which
+give vector magnitude by the length of the arrow, the barbs give more
+quantitative information about the vector magnitude by putting slanted
+lines or a triangle for various increments in magnitude, as show
+schematically below::
+
+  :                   /\
+  :                  /  \
```

Comment:
Need to maintain the full height schematic line as well (the one 4 \ high) if we're going to maintain my crappy ascii art.